dev.vencord.Vesktop.metainfo.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <!--Created with jdAppStreamEdit 7.1-->
+  <id>dev.vencord.Vesktop</id>
+  <name>Vesktop</name>
+  <summary>Vesktop is a cross platform desktop app aiming to give you a snappier Discord experience with Vencord pre-installed.</summary>
```

Comment:
```suggestion
  <summary>Vesktop is a cross platform desktop app aiming to give you a snappier Discord experience with Vencord pre-installed</summary>
```
https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/quality-guidelines#no-weird-formatting-1

Reply:
"Snappier Discord app/experience" is better https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/quality-guidelines#not-technical